components/match2/commons/match_group_input.lua

```diff
@@ -813,49 +838,67 @@ function MatchGroupInput.getMapVeto(match, allowedVetoes)
 	return data
 end
 
----@param opponents table[]
+---@param opponents {status: string?, score: any}[]
 ---@return boolean
 function MatchGroupInput.isDraw(opponents)
 	if Logic.isEmpty(opponents) then return true end
```

Comment:
i know this line doesn't get changed but i still think applying draw for matches without opponents isn't ideal

Reply:
I agree

Reply:
#4268 for reference

Reply:
when refactoring everything we maybe should prep all wikis to change the behavior (oos for this pr though)

Reply:
when refactoring everything we maybe should prep all wikis to change the behavior (oos for this pr though)

Reply:
indeed